Real Analysis/real-analysis-notes.tex

```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
   third coordinates. In other words, this set is in bijection with the
   set $\P(\{0,1\}^2)$.
 \end{example}
+\subsection*{(1/23/2017) Lecture 2}
+\begin{example}[Product $\sigma$-algebra]
+  Let $(x_\alpha)_{\alpha \in A}$ be an indexed family of sets as
+  described above. Then, for each $\alpha \in A$, let $M_\alpha$ be a
+  $\sigma$-algebra on $X_\alpha$ given by \[
+    X = \prod_{\alpha \in A} X_\alpha
+  \]
+  We take $f_\alpha = \pi_\alpha$, that is, the projection function on
+  to $\alpha$ given by \[
+    \pi_\alpha: X \to X_\alpha 
+  \] \[
+    (x_\beta)_{\beta \in A} \mapsto x_\alpha.
+  \]
+  This gives a $\sigma$-algebra on $X$ called the \emph{product
+    $\sigma$-algebra}, denoted \[
+    \bigotimes_{\alpha \in A} \M_\alpha.
+  \]
+  In more concise terms, we say that \[
```

Comment:
I disagree. The previous part is a definition that specifies notation, the latter part is a very condensed mathematical expression. I can rework it to make the first part more related to definition 1.12.